data/hai missions.txt

```diff
@@ -13,6 +13,7 @@ mission "First Contact: Hai"
 	source
 		attributes "hai"
 	on offer
+		log "Made contact with a species north of human space, through a wormhole. They call themselves the Hai, and seem to be peaceful. But they warn of renegades in the north of their space."
```

Comment:
Maybe don't mention the wormhole since the player may not have used it to reach them (jump drives are fun).
Combine the last two sentences: `peaceful. But` -> `peaceful, but`. I'd change "in the north of" to "in the northern region of," but that might just be preference.